coralogix-policies/coralogix-metrics-integration-policy/CHANGELOG.md

```diff
@@ -2,34 +2,38 @@
 
 ## AwsMetrics
 
-### 18.10.2024
-### Fix rds:ListTagsForResource permission
+### 5.12.2024 New permissions, that would allow integration to get data from Amazon ElastiCache API
```

Comment:
I think that needs to change the date for today. or is it a different change?

Reply:
yeah sure